cmd/gossamer/config.go

```diff
@@ -433,6 +434,8 @@ func setDotGlobalConfig(ctx *cli.Context, tomlCfg *ctoml.Config, cfg *dot.Global
 		cfg.MetricsPort = uint32(metricsPort)
 	}
 
+	cfg.NoTelemetry = ctx.Bool("no-telemetry")
```

Comment:
Instead of `no-telemetry` wouldn't it be better if the flag is named `telemetry`?

Reply:
I'm replicating the behavior of the paritytech polkadot node, which has telemetry enabled by default, and a `no-telemetry` flag to disable this.